### ISSUE TITLE
image_cropper dependency causes compile error

### ISSUE THREAD

**Issue by StaticReturn** [jayeshpansheriya/image_picker_type#5]
image_cropper 1.5.0 has a change that causes the compile to fail: BREAKING CHANGE: change cropImage() to instance method.

The error is as follows.

```
: Error: Member not found: 'ImageCropper.cropImage'.
../…/lib/image_picker_type.dart:66
      return ImageCropper.cropImage(
                          ^^^^^^^^^
2

FAILURE: Build failed with an exception.
```

**Comment by jayeshpansheriya** [comment]
@StaticReturn  try latest version

**Comment by jayeshpansheriya** [comment]
Resolved in `1.0.6`, add web support in `1.1.0`
